src/manager/orgsAndUsers.js

```diff
@@ -206,7 +206,7 @@ export async function usersInOrg(orgId) {
 // same as above, but exclude me
 export async function inMyOrg(organization, me) {
   if (!organization.id) return [];
-  const users = await getUsers({ orgIds: [organization.id] });
+  const users = await getUsers({ orgIds: [organization.id] }).catch((e) => []);
```

Comment:
I think the problem with swallowing the error is that we'll have a hard time knowing that there's a problem if one is happening. If we don't have an established error handling pattern in place (and I think we might not, unfortunately; something like Sentry would be great) maybe it's best to log with `console.error` so we could at least see it ourselves when trying to reproduce weird behavior in the future.